src/popup/pages/SecureLoginSettings.vue

```diff
@@ -1,124 +1,267 @@
 <template>
   <IonPage>
     <IonContent class="ion-padding ion-content-bg">
-      <div
-        v-if="IS_MOBILE_APP"
-        class="secure-login-settings"
-      >
-        <div class="top-wrapper">
+      <div class="secure-login-settings">
+        <div
+          v-if="IS_MOBILE_APP"
+          class="top-wrapper"
+        >
           <p class="text-description">
             {{ $t('pages.secureLogin.description') }}
           </p>
 
           <SwitchButton
             :label="$t('pages.secureLogin.enableSecureLogin')"
-            :model-value="isSecureLoginEnabled"
-            :disabled="!isSecureLoginAvailable"
-            @update:modelValue="setSecureLoginEnabled"
+            :model-value="isBiometricLoginEnabled"
+            :disabled="!isBiometricLoginAvailable"
+            @update:modelValue="setBiometricLoginEnabled"
           />
         </div>
-        <hr>
+        <hr v-if="IS_MOBILE_APP">
         <div class="options-wrapper">
           <div class="options">
             <div
               class="options-info"
-              :class="{ dimmed: !isSecureLoginEnabled }"
+              :class="{ dimmed: IS_MOBILE_APP && !isBiometricLoginEnabled }"
             >
-              <span class="options-label">
-                {{ $t('pages.secureLogin.autoLock.title') }}
-              </span>
-              <span class="options-description">
-                {{ $t('pages.secureLogin.autoLock.descriptionPart1') }}
-              </span>
-              <span class="options-description">
-                {{ $t('pages.secureLogin.autoLock.descriptionPart2') }}
-              </span>
+              <span class="options-label" v-text="$t('pages.secureLogin.autoLock.title')" />
+              <span class="options-description" v-text="$t('pages.secureLogin.autoLock.descriptionPart1')" />
+              <span class="options-description" v-text="$t('pages.secureLogin.autoLock.descriptionPart2')" />
             </div>
 
             <RadioButton
               v-for="(ms, minutes) in AUTHENTICATION_TIMEOUTS"
               :key="`timeout-${minutes}`"
               :value="secureLoginTimeout === ms"
-              :disabled="!isSecureLoginEnabled"
               :class="{ active: secureLoginTimeout === ms }"
               class="timeout"
               has-label-effect
               @input="setSecureLoginTimeout(ms)"
             >
-              <div class="row">
-                {{ $t('pages.secureLogin.authenticationTimeout', Number(minutes)) }}
-              </div>
+              <div class="row" v-text="$t('pages.secureLogin.authenticationTimeout', Number(minutes))" />
             </RadioButton>
           </div>
         </div>
-      </div>
+        <template v-if="!IS_MOBILE_APP">
```

Comment:
`v-else` would probably  be better choice as we already check `v-if="IS-MOBILE_APP"` at the top.

Reply:
`v-else` would not work here because the template is not directly after the `div` with `v-if="IS-MOBILE_APP"`